resources/css/navigation.css

```diff
@@ -0,0 +1,27 @@
+@tailwind base;
+ 
+@tailwind components;
+ 
+@tailwind utilities;
+
+.header{
+	
+}
+.siteLogo{
+	max-width: 100px;
+}
+
+.navigationBar{
+	font: 15px Arial, sans-serif;
+	vertical-align: top;
+}
+.navItems{
```

Comment:
Style: include an empty line between the classes

Reply:
added